README.md

```diff
@@ -65,9 +65,56 @@ recommended Java logging API within Google.
 
 ### 1. Add the dependencies on Flogger
 
-All code that uses flogger should depend on
-`com.google.flogger:flogger:<version>` and
-`com.google.flogger:flogger-system-backend:<version>`.
+All code that uses flogger should depend on:
+
+* `com.google.flogger:flogger:<version>`
+* `com.google.flogger:flogger-system-backend:<version>`
+
+**Maven**
+
+```xml
+    <dependency>
+      <groupId>com.google.flogger</groupId>
+      <artifactId>flogger</artifactId>
+      <version>${flogger.version}</version>
+    </dependency>
+    <dependency>
```

Comment:
It might be helpful to move the NOTE from below to *above* the Maven subheading and add inline XML comments like:

<!-- only in root component of your application --> 

above `flogger-log4j-backend`